### DISCUSSION TITLE
Connect to MQTT power meter

### DISCUSSION THREAD

**Discussion by gibzwein** [hoylabs/OpenDTU-OnBattery#1099] [Q&A] [answered]
Hi,
I have an issue to connect to my power meter over the MQTT.
I can do this easily via MQTT Explorer:
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/049ed7a4-953d-4bad-98c1-409808a6de7d)
but in OpenDTU i have:
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/501d83f7-80ab-4a08-aea1-fd2334fcb3c6)
These are my settings:
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/299e96d2-d209-455c-a601-9309c0d53ebb)
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/32103980-6a2c-4b6d-b49a-5023b18eda21)
What am I doing wrong?
Best regards

**Comment by schlimmchen** [comment]
The connection to the broker cannot be established. Are the port, username, and password settings correct?

**Comment by gibzwein** [comment]
I did it many times.
I dont know the issue:
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/8041a9ff-5f86-4d3a-9241-9c8aacb2ce5c)

**Reply by schlimmchen** [reply]
TLS is enabled here, whereas you disabled it in OpenDTU-OnBattery.

**Reply by gibzwein** [reply, marked as answer]
I tried with and without TLS, also paste another KEY.
OK I pasted KEY from supla website and it looks like working
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/27d5af1b-d2e9-4341-bbe1-8377d15d7255)
but i still have 0 zeroes
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/dac1a9fe-3d64-4c3f-8632-885c9a44a53c)
I added supla at the begging of:
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/f3ac43b0-cb03-41c3-ad13-93119aba87f8)
And now it works:
![obraz](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/52330511/48af42cc-3d40-4b6b-a57e-08d10aa9ef16)
Thanks

**Comment by gibzwein** [comment]
I see in description that "You cannot add any device" to supla broker, so maybe this is the issue?
